xcodeproj/internal/resources.bzl

```diff
@@ -131,9 +131,6 @@ def _add_structured_resources_to_bundle(
         )
         bundle.resources.append(fp)
 
-        if not file.is_source:
-            extra_files.append(file_path(file, include_in_navigator = False))
```

Comment:
Linter saying `extra_files` is not being used anymore, you'll probably have to remove from call sites of `_add_structured_resources_to_bundle` too